src/ActionModal/components/ActionModal.vue

```diff
@@ -812,10 +821,12 @@ export default {
         ),
         // we don't use SMALLEST as min gas price because it can be a fraction of uatom
         // min is 0 because we support sending 0 fees
-        between: between(0, this.selectedBalance.amount)
+        max: x => Number(x) <= this.selectedBalance.amount,
+        min: x => Number(x) >= 0
       },
       invoiceTotal: {
-        between: between(0, this.selectedBalance.amount)
+        max: x => Number(x) <= this.selectedBalance.amount,
+        min: x => Number(x) >= SMALLEST
```

Comment:
does this make sense?

Reply:
yes

Reply:
hahahah why? can you help me understand with an example?

Reply:
I see! negative numbers